cmd/protoc-gen-fastmarshal/funcs.go

```diff
@@ -204,6 +204,10 @@ func addProtoFunctions(fm template.FuncMap, protoFile *protogen.File) template.F
 	fm["getImportPrefix"] = getImportPrefix(protoFile)
 	fm["mapFieldGoType"] = mapFieldGoType(protoFile)
 	fm["hasRequiredFields"] = hasRequiredFields(protoFile)
+	fm["is_gogo_nullable"] = makeMapFieldOption(protoFile, "65001:1")
```

Comment:
These feel very "magic" and I don't see them used.  What's the intent here?

For context, the "point" of this library is to decouple from Gogo so I'm against adding something Gogo-specific in general.